res/analytics.js

```diff
@@ -9,7 +9,7 @@
     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```

Comment:
without this change, CSP blocks the "http" version in the dev environment. 2 possible fixes: either this, or adding the "http" version to the CSP header  -- which redirects to https anyway. So I think it's better to always ask for the https version, in this case.